modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/DropdownMenu.js

```diff
@@ -26,7 +26,20 @@ export default function NavigationBar({
 }) {
 	return (
 		<ClayDropDownWithItems
-			items={items}
+			items={items.map(({data, ...rest}) => {
+				const dataAttributes = data
+					? Object.keys(data).reduce((acc, attribute) => {
```

Comment:
`Object.entries` would be better than `Object.keys`, because you need both the key *and* the value here.

Reply:
Indeed!

Reply:
Done at https://github.com/liferay-frontend/liferay-portal/pull/2/commits/95e11e7832bbff3e538f91faab505d40f31ca2b3#diff-f0525bff0b549a2b86d74130c53122e1R35